starter-code/views/index.ejs

```diff
@@ -5,6 +5,6 @@
     <link rel="stylesheet" type="css/text" href="../public/stylesheets/style.css">
   </head>
   <body>
-
+    <p></p>
```

Comment:
You have to reference here the key of the object you passed (line 23 of app.js):
```
<p><%= joke %></p>
```